Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -28,7 +29,8 @@ def contributions_args
       comma_array "--repositories",
                   description: "Specify a comma-separated (no spaces) list of repositories to search. " \
                                "Supported repositories: #{SUPPORTED_REPOS.map { |t| "`#{t}`" }.to_sentence}." \
-                               "Omitting this flag, or specifying `--repositories=all`, will search all repositories."
+                               "Omitting this flag, or specifying `--repositories=all`, searches all repositories." \
+                               "Use `--repositories=primary` to search only the main repositories: brew,core,cask."
```

Comment:
Nit: Would it better to make this a separate option like `--primary`?

Reply:
That might be nice but I'm fine either way!